css/styles.css

```
@@ -223,19 +203,33 @@
   }
 
   .advantages-list-inside {
-    display: flex;
+    display: grid;
+    grid-template-columns: 400px 400px 400px;
```

Лучше вместо пикселей задать относительную величину:
```suggestion
    grid-template-columns: 1fr 1fr 1fr;
```